packages/web3/README.md

```diff
@@ -101,4 +101,4 @@ The `<NFTBalance />` component allows you to display all the NFT's an address ow
 
 If you want to use this component wrap your app with `<MoralisProvider appId={MORALIS_APP_ID} serverUrl={MORALIS_SERVER_URL} ></MoralisProvider >`. Read more about moralis [here](https://docs.moralis.io/introduction/readme)
 
-Try the `<NFTBalance />` component in the [interactive documentation](http://localhost:6006/?path=/docs/1-web3-nft-balance--default)
+Try the `<NFTBalance />` component in the [interactive documentation](http://localhost:6006/?path=/docs/1-web3-parse-nft-balance--default)
```

Comment:
This link points to localhost right now...

Could you also update this link and make it similar to other components?

Reply:
okay I do it !

Reply:
done